lib/simulatelocation/index.js

```diff
@@ -0,0 +1,51 @@
+//We just need to write any data to the client. It doesn't matter what we send
+const SIMULATE_LOCATION_SERVICE_NAME = 'com.apple.dt.simulatelocation';
+
+const RESET_MESSAGE = Buffer.from([0, 0, 0, 1]);
+const SET_LOCATION_MESSAGE = Buffer.from([0, 0, 0, 0]);
+
+class SimulateLocationService {
```

Comment:
LocationSimulatorService

Reply:
Does it make a difference? I mean the service on the phone side is called simulatelocation